packages/zowe-explorer-ftp-extension/README.md

```diff
@@ -39,14 +47,46 @@ The following steps are not required, as the Zowe Explorer FTP extension also in
    zowe plugins install @zowe/zos-ftp-for-zowe-cli@latest
    ```
 
-1. Create Zowe FTP profile:
+2. Create Zowe FTP profile:
 
    ```bash
    zowe profiles create zftp <profile name> -H <host> -u <user> -p <password> -P <port>
    ```
 
 Now you can run `zowe zos-ftp` commands as documented in the docs for the plugin. This profile can then also be selected in Zowe Explorer's Add Profile dialogs once this Zowe Explorer FTP VS Code extension is installed.
 
+## List of Supported Data Set Functionalities
+
+See the list of the supported functionalities for different types of data sets:
+
+Migrated Dataset:
+
+- Show Data Set Attribute — Supported
+- Add to Favorites — Supported
+
+Sequential Dataset:
+
+- Show Data Set Attribute — Supported
```

Comment:
Perhaps we can remove the `- Supported` tag for each of them since we only list supported ones anyway